packages/terra-application-utility/src/header/ApplicationHeaderUtility.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import IconChevronDown from 'terra-icon/lib/icon/IconChevronDown';
+import UtilityMenu from './_HeaderUtilityMenu';
+import Utils from '../_Utils';
+import styles from './ApplicationHeaderUtility.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The array containing the menu items to be rendered within the menu.
+   */
+  menuItems: PropTypes.arrayOf((Utils.itemShape)).isRequired,
+  /**
+   * The function to trigger when a menu item is selected.
+   */
+  onChange: PropTypes.func.isRequired,
+    /**
+   * The function that discloses the menu.
+   */
+  onDisclose: PropTypes.func.isRequired,
+  /**
+   * The name to be displayed next to the user photo.
+   */
+  userName: PropTypes.string,
```

Comment:
Based on @dkasper-was-taken comment regarding application housing the UserData component, should the `userName` and `userPhoto` props be removed in favor of a `userItem` prop? Then expect an Avatar or ProfileImage component to be passed in? Then we could remove the baked-in rendering if no user props are provided?

Reply:
I am wondering if we should change these props to be `content` or `title` and `accessory`? For example, if consuming this in site, I would want a the Utilities icon to display and the `userPhoto` prop feels misleading.

Reply:
Yeah, UX callouts point to Utilities being directly associated with user data. But, I can see how making the props generic makes more sense, especially since Utilities should now be agnostic on what is being passed in.